stories/core/Tabs.stories.tsx

```diff
@@ -50,7 +57,7 @@ DefaultTabs.args = {
   ],
 };
 
-export const BorderlessTabs = Template.bind({});
+export const BorderlessTabs = DefaultTabs.bind({});
 BorderlessTabs.args = {
   labels: [
```

Comment:
Can you remove `labels`?

Reply:
🤔 It will not work if I do. The story does not have labels.

Reply:
Shouldn't it take them from `DefaultTabs`?

Reply:
tested and saw 😀